File: src/main/java/io/github/dylmeadows/eontimer/model/timer/TimerConstants.java
package io.github.dylmeadows.eontimer.model.timer;

import lombok.AccessLevel;
import lombok.NoArgsConstructor;

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class TimerConstants {
    public static final double UPDATE_FACTOR = 1.0;
    public static final double CLOSE_UPDATE_FACTOR = 0.75;
    public static final int CLOSE_THRESHOLD = 167;
    public static final long NULL_TIME_SPAN = -999;
    public static final long INFINITE_TIME_SPAN = -99;
    public static final TimerType DEFAULT_TIMER_TYPE = TimerType.GEN5;
}

File: src/main/kotlin/io/github/dylmeadows/eontimer/model/timer/TimerConstants.kt
package io.github.dylmeadows.eontimer.model.timer

object TimerConstants {
    @JvmField
    val DEFAULT_TIMER_TYPE = TimerType.GEN5
    const val UPDATE_FACTOR = 1.0
    const val CLOSE_UPDATE_FACTOR = 0.75
    const val CLOSE_THRESHOLD = 167
    const val NULL_TIME_SPAN = -999L
    const val NULL_TIME_SPAN_TEXT = "0:00"
    const val INFINITE_TIME_SPAN = -99L
    const val INFINITE_TIME_SPAN_TEXT = "?:??"
}